de.wwu.md2.framework/src/de/wwu/md2/framework/generator/backend/EnumAndEntityClass.xtend

```diff
@@ -288,13 +294,61 @@ class EnumAndEntityClass {
                 return requestMethod;
             }
         
-            public HashMap<String, Object> getQueryParams() {
+            public List<CustomHashMapEntry> getQueryParams() {
                 return queryParams;
             }
         
-            public HashMap<String, Object> getBody() {
+            public List<CustomHashMapEntry> getBody() {
                 return body;
             }
+            
+                
+            public void setUrl(String url){
+                this.url = url;
+            }
+
+        public void setRequestMethod(RequestMethod method){
+        	this.requestMethod = method;
+        }    
+
+        public void setQueryParams(List<CustomHashMapEntry> queryparams){
+        	this.queryParams = queryparams;
+        }    
+
+        public void setBody(List<CustomHashMapEntry> body){
+        	this.body = body;
+        }
+
+        @XmlRootElement
+        public static class CustomHashMapEntry {
+        
+        	@XmlElement
+            public String key; 
+
+        	@XmlElement
+            public String value;
```

Comment:
Because "Object" caused "value" to contain a JSONValue (or so) object, 
which could not be used properly by the rest of the web service.